File: Lógica/Lista2/ExPySala/ex5.py
valorA = float(input("Digite o valor A: "))
valorB = float(input("Digite o valor B: "))
valorC = float(input("Digite o valor C: "))

if (valorA > valorB and valorA> valorC and (valorB <= valorC)):
    print(f'{valorB}, {valorC}, {valorA}')
elif (valorA > valorB and valorA> valorC and (valorC <= valorB)):
    print(f'{valorC}, {valorB}, {valorA}')
elif (valorB > valorC and valorB > valorA and (valorC <= valorA)):
    print(f'{valorC}, {valorA}, {valorB}')
elif (valorB > valorC and valorB > valorA and (valorA <= valorC)):
    print(f'{valorA}, {valorC}, {valorB}')
elif (valorC > valorA and valorC > valorB and (valorB <= valorA)):
    print(f'{valorB}, {valorA}, {valorC}')
elif (valorC > valorA and valorC > valorB and (valorA <= valorB)):
    print(f'{valorA}, {valorB}, {valorC}')

File: Lógica/Lista2/ExPySalaC/ex5.c
/// Código Apenas usando if e else 

#include <stdio.h>

void main() {
    float valorA, valorB, valorC;

    printf("Digite o valor A: ");
    scanf("%f", &valorA);
    printf("Digite o valor B: ");
    scanf("%f", &valorB);
    printf("Digite o valor C: ");
    scanf("%f", &valorC);

    if (valorA > valorB && valorA > valorC && (valorB <= valorC)) {
        printf("%.2f, %.2f, %.2f\n", valorB, valorC, valorA);
    } 
    else if (valorA > valorB && valorA > valorC && (valorC <= valorB)) {
        printf("%.2f, %.2f, %.2f\n", valorC, valorB, valorA);
    } 
    else if (valorB > valorC && valorB > valorA && (valorC <= valorA)) {
        printf("%.2f, %.2f, %.2f\n", valorC, valorA, valorB);
    } 
    else if (valorB > valorC && valorB > valorA && (valorA <= valorC)) {
        printf("%.2f, %.2f, %.2f\n", valorA, valorC, valorB);
    } 
    else if (valorC > valorA && valorC > valorB && (valorB <= valorA)) {
        printf("%.2f, %.2f, %.2f\n", valorB, valorA, valorC);
    } 
    else if (valorC > valorA && valorC > valorB && (valorA <= valorB)) {
        printf("%.2f, %.2f, %.2f\n", valorA, valorB, valorC);
    }

}
